apps/extension/src/features/trpc/api/provider.tsx

```diff
@@ -12,7 +13,13 @@ export const TRPCProvider: React.FC<{ children: React.ReactNode }> = ({
   const [queryClient] = React.useState(() => new QueryClient())
   const [trpcClient] = React.useState(() =>
     trpcApi.createClient({
+      transformer,
       links: [
+        loggerLink({
+          enabled: (opts) =>
+            process.env.NODE_ENV === "development" ||
+            (opts.direction === "down" && opts.result instanceof Error)
```

Comment:
What's direction === "down"?

I like the instanceof Error boolean. 👍

Reply:
down is an incoming request, up would be outgoing.

So this would log in dev, or for incoming errors

Reply:
Thanks, TIL!